spec/src/modules/tracker.js

```diff
@@ -2077,71 +2079,7 @@ describe(`ConstructorIO - Tracker${bundledDescriptionSuffix}`, () => {
       });
 
       // eslint-disable-next-line max-len
-      expect(tracker.trackSearchResultsLoadedV2(term, { ...requiredParameters, ...v2Parameters })).to.equal(true);
-    });
-
-    it('Backwards Compatibility - Should respond with a valid response when term and snake cased parameters are provided', (done) => {
```

Comment:
I think I might've commented on this in the old PR as well, but should we keep this test for snake cased params, around?

Reply:
Yea you did 😂 . I think the test is still there. We just had separate ones for v1/v2 before so we deleted one of them.